subsys/llext/Kconfig

```
@@ -44,8 +44,21 @@ config LLEXT_TYPE_ELF_SHAREDLIB
 
 endchoice
 
+config LLEXT_HEAP_DYNAMIC
+	bool "Do not allocate static LLEXT heap"
+	help
+	  Some applications require loading extensions into the memory which does not
+	  exist during the boot time and cannot be allocated statically. Make the application
+	  responsible for LLEXT heap allocation. Do not allocate LLEXT heap statically.
+
+	  Application must call llext_heap_init() in order to allocate the LLEXT heap memory.
+	  If llext_heap_init() is not called an LLEXT module will not load. When the application
+	  is finished with the LLEXT loader it should call llext_heap_uninit(), which does not
+	  free the memory, but marks it insaccessible by the LLEXT loader any more.
```

Tried to avoid using "allocate" since that is the app's responsibilty. 
```suggestion
	  Application must call llext_heap_init() in order to assign a buffer to be used
	  as the LLEXT heap, otherwise LLEXT modules will not load. When the application
	  does not need LLEXT functionality any more, it should call llext_heap_uninit(),
	  which releases control of the buffer back to the application.
```